index.html

```diff
@@ -9,30 +9,46 @@
                 
 </head>
 <body>
+    <section id="main-page">
      <header class="nav-bar">
                 <a class="logo" href="#">Muhammad Nafees</a>
-                <nav class="menu" href="#"><img src="images/Union.png" alt="Menu bar"></nav>
+                <nav class="menu" href="#"><img id="menu-icon" src="images/menu.png" alt="Menu bar">
+                <div id="nav-menu">
+                    <a href="#">Portfolio</a>
+                    <a href="">About</a>
+                    <a href="">Contact</a>
+                    <a href=""><img src="images/mail-icon.png" alt=""></a>
+                </div></nav>
      </header>
         <section class="content">
+            <div class="main-text">
     <h1 class="my-name">Hey There.<br/> I'm Nafees</h1>
     <h3 class="profession">I’m a Software Developer</h3>
     <p class="i-can-do">I can help you build a product , <br /> feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, 
         don’t hesitate to contact me.</p>
+    </div>
         <ul class="links">
             <li><a title="GitHub" href="https://github.com/ngeelani48"><img src="images/github.png" alt="GitHub Profile"></a></li>
             <li><a title="Twitter" href="https://twitter.com/ngeelani48"><img src="images/twitter.png" alt="Twitter Profile"></a></li>
             <li><a title="Medium" href="https://medium.com/@engnrnafees"><img src="images/medium.png" alt="Medium Profile"></a></li>
             <li><a title="LinkedIn" href="https://www.linkedin.com/in/m-n-geelani-3446097275/"><img src="images/Linkedin icon.png" alt="LinkedIn Profile"></a></li>
             <li><a title="Angelist" href="#"><img src="images/Vector.png" alt="angelist Profile"></a></li>
-        </ul>
+        </ul>  
     </section>
+</section>
     <section class="project-section">
+        <div class="my-recent-works">
         <div class="my-work-title">
             <h1>My Recent Works</h1>
         </div>
         <div id="line"></div>
+    </div>
         <div id="works-section">
```

Comment:
- [OPTIONAL] I think it would be better if you fix your work cards just a little bit. While moving from mobile version to desktop version, these cards are slightly miss-placed. How about fixing the following? What do you think about making these cards responsive as well? 🎯 You are almost there 🏁

## **_Your design 👇_**
![image](https://user-images.githubusercontent.com/106119105/213634302-abfa2205-d267-4978-9acf-5ce754dd1b2b.png)


## **_Figma design 👇_**
![image](https://user-images.githubusercontent.com/106119105/213634423-f411d8a3-462e-4bc7-838e-6a4c08b1f532.png)